devel/git-latexdiff/Portfile

```diff
@@ -18,11 +18,12 @@ long_description    Tool to graphically visualize differences between \
                     and latexdiff.
 
 homepage            https://gitlab.com/${name}/${name}
-master_sites        https://gitlab.com/${name}/${name}/repository/archive.tar.gz?ref=v${version}&dummy=
-distname            ${name}-v${version}-2bd0dcbf2a14de7c311382ed03eb24bac661ac69
+master_sites        https://gitlab.com/${name}/${name}/-/archive/v${version}
+distname            ${name}-v${version}
 
-checksums           rmd160  29b61b526bbed34d32ed8d945c3e569a36d10fc7 \
-                    sha256  76b19b623749025bde0617776237dbd6763a21d19cd6b6b353765846dac766ec
+
+checksums           rmd160  9e89a7973114af83daff586b76740f44912b71df \
+                    sha256  54ba32ca75feef37a8d47b7b35d29a82819031a408bdb3eb25c1da73e68811e4
```

Comment:
Please add a `size` field to `checksums` and we will merge this.

Reply:
Where do I find an example or documentation on the `size` field?

Reply:
It's not documented yet. But you can look at another port with it such as py-numpy. You can determine the correct value by initially setting it to 1 and then run `sudo port -v checksum git-latexdiff` to get the correct value.

Reply:
@mf2k I was unaware it wasn't currently in the guide. Could you add it? I'd do it but I've got limited time ATM.